tools/clang/test/DXC/denorm.test

```diff
@@ -1,9 +1,15 @@
 // Test for denorm options
 // RUN: %dxc  %S/Inputs/smoke.hlsl /Tps_6_2 /denorm preserve | FileCheck %s
 // CHECK:"fp32-denorm-mode"="preserve"
+// RUN: %dxc  %S/Inputs/smoke.hlsl /Tps_6_2 | FileCheck %s --check-prefix=CLEAN
+// CLEAN-NOT:"fp32-denorm-mode"
+// CLEAN:define void @main() 
```

Comment:
I don’t think this works the way you think it does. A `NOT` directive just means it can’t be found until the next `CHECK` directive. This verifies that the attribute doesn’t occur before the main declaration, but the attribute list is always later in the file the function declarations.

A check on something early in the file (like `datalayout`, which is always early) followed by the `NOT` line will verify that the string doesn’t occur before the end of the file.